pkg/scheduler/eventhandlers.go

```diff
@@ -225,7 +237,19 @@ func (sched *Scheduler) updatePodInCache(oldObj, newObj interface{}) {
 		logger.Error(err, "Scheduler cache UpdatePod failed", "pod", klog.KObj(oldPod))
 	}
 
-	sched.SchedulingQueue.AssignedPodUpdated(logger, oldPod, newPod)
+	// SchedulingQueue.AssignedPodUpdated internally pre-filters Pods to move to activeQ while taking only in-tree plugins into consideration.
+	// Consequently, if custom plugins that subscribes Pod/Update events reject Pods,
```

Comment:
Can we make the comment more concise?  It's a little too long.

Reply:
So, this PR solves two things:
- take PodTopologySpread into consideration when requeueing Pods based on Pod related events.
  - This fix is mostly for people who don't use QHint enabled.
- use MoveAllToActiveOrBackoffQueue only, when the feature gate is enabled.
  - It's the general fix to fix the same problem in custom plugins. We have to guard it with feature flag, otherwise negative impact on the throughput. (it's like we can't remove preCheck until QHint is done)

Maybe you want me to create separate PR for the changes in eventhandlers.go?

Reply:
Updated the description.